src/backends/ncnn/ncnnlib.cc

```diff
@@ -116,8 +116,20 @@ namespace dd
         } catch (...) {
             throw;
         }
+        if (_old_height != -1 && _old_height != inputc.height())
+          {
+            // in case of new prediction with different timesteps value, clear old net
```

Comment:
Is there any hope of not re-creating the net ? Do we have an potential example of time-series in prediction with variable time-step ?

Reply:
the net is re created only in case of change of height / timesteps. Generally speaking it will be created only once: is is created only in here, not at init_mllib. Here we just check in case height / timestep has changed (which can happen especially during debug :) .